pubsublite/consumer.go

```diff
@@ -171,6 +175,11 @@ func NewConsumer(ctx context.Context, cfg ConsumerConfig) (*Consumer, error) {
 				zap.String("region", subscription.Region),
 				zap.String("project", subscription.Project),
 			),
+			telemetryAttributes: []attribute.KeyValue{
+				semconv.MessagingSourceNameKey.String(subscription.Name),
+				attribute.String("region", subscription.Region),
```

Comment:
Shouldn't this be using [`semconv.CloudRegion`](https://pkg.go.dev/go.opentelemetry.io/otel@v1.14.0/semconv/v1.18.0#CloudRegion)? 

```suggestion
				semconv.CloudRegion(subscription.Region),
```